oauth2/fosite_store_sql.go

```diff
@@ -163,9 +174,26 @@ func sqlSchemaFromRequest(signature string, r fosite.Requester, logger logrus.Fi
 	}, nil
 }
 
-func (s *sqlData) toRequest(session fosite.Session, cm client.Manager, logger logrus.FieldLogger) (*fosite.Request, error) {
+// guess if string json or encrypted string.
+func maybeJSONObj(s string) bool {
+	if s == "null" {
+		return true
+	}
+	return strings.HasPrefix(s, "{") && strings.HasSuffix(s, "}")
+}
+
+func (s *sqlData) toRequest(session fosite.Session, cm client.Manager, conf Configuration, kc *jwk.AEAD, logger logrus.FieldLogger) (*fosite.Request, error) {
+	sess := s.Session
+	if !maybeJSONObj(string(sess)) {
```

Comment:
I don't think we're using this library here but gjson is a really good library and has a very fast `gjson.Valid` function that works with all JSON types: https://github.com/tidwall/gjson/blob/master/gjson.go#L2470

```suggestion
	if !gjson.ValidBytes(sess) {
```

Reply:
This lib looks pretty good. Thank you.